style.css

```diff
@@ -29,6 +29,11 @@ body {
   --slider-height: calc(var(--switch-height) - var(--switch-padding));
   --slider-on: calc(var(--switch-height) - var(--switch-padding));
 }
+
+.power-switch {
+  margin: 30px;
```

Comment:
this margin does not match the original

Reply:
<div class="power-switch" style="margin: 3px;">